config/environments/production.rb

```diff
@@ -98,6 +98,7 @@
   #   /.*\.example\.com/ # Allow requests from subdomains like `www.example.com`
   # ]
   config.hosts = ENV["PRODUCTION_HOST"].split(",").map(&:strip) if ENV["PRODUCTION_HOST"]
+  # config.hosts << Setting.app_url_host
```

Comment:
Why does development add this to `config.hosts` but production not add it?

Reply:
ah I meant to uncomment this one. Where is PRODUCTION_HOST being created? Maybe it would be better to combine them? Should probably do the same with the DEV_HOST env too?